spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+RSpec.describe FeedbacksHelper, type: :helper do
+  describe 'pending_feedback_link' do
+    let(:pair_request_author) { create(:user) }
+    let(:pair_request_invitee) { create(:user) }
+    let(:random_user) { create(:user) }
+    
+    let(:pair_request) { create(:pair_request, :completed_with_feedback, author_id: pair_request_author.id, invitee_id: pair_request_invitee.id)}
+
+    let(:author_feedback) { pair_request.author_feedback }
+    let(:invitee_feedback) { pair_request.invitee_feedback }
+ 
+    context "when current_user == feedback.author and pair_request status: :draft" do
```

Comment:
write these as like `when current_user is the author and the pair_request is a draft` do